Python/dp/N-Queen.py

```diff
@@ -0,0 +1,91 @@
+class QueenChessBoard:
+    def __init__(self, size):
+        self.size = size
+        self.columns = []
+        self.count = 0
+ 
+    def place_in_next_row(self, column):
+        self.columns.append(column)
+ 
+    def remove_in_current_row(self):
+        return self.columns.pop()
+ 
+    def is_this_column_safe_in_next_row(self, column):
```

Comment:
Use docstrigs to explain what each function does.

Reply:
Done